docs/rules/void-dom-elements-no-children.md

```diff
@@ -1,4 +1,4 @@
-# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children
+# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children (react/void-dom-elements-no-children)
```

Comment:
We did not put the `react/` prefix in the other titles, I think it's not necessary to put it here either